src/main/java/com/altenheim/kalender/controller/logicController/ImportController.java

```diff
@@ -1,8 +1,38 @@
 package com.altenheim.kalender.controller.logicController;
 
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.text.ParseException;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.util.GregorianCalendar;
+
 import com.altenheim.kalender.interfaces.ICalendarEntriesModel;
 import com.altenheim.kalender.interfaces.IImportController;
 import com.altenheim.kalender.interfaces.IWebsiteScraperController;
+import com.calendarfx.model.Entry;
+
+import net.fortuna.ical4j.data.CalendarBuilder;
+import net.fortuna.ical4j.data.CalendarOutputter;
+import net.fortuna.ical4j.data.ParserException;
+import net.fortuna.ical4j.model.Calendar;
+import net.fortuna.ical4j.model.ComponentList;
+import net.fortuna.ical4j.model.DateTime;
+import net.fortuna.ical4j.model.Property;
+import net.fortuna.ical4j.model.component.CalendarComponent;
+import net.fortuna.ical4j.model.component.VEvent;
+import net.fortuna.ical4j.model.property.CalScale;
+import net.fortuna.ical4j.model.property.DtEnd;
+import net.fortuna.ical4j.model.property.DtStart;
+import net.fortuna.ical4j.model.property.ProdId;
+import net.fortuna.ical4j.model.property.Version;
+import net.fortuna.ical4j.model.property.XProperty;
+import net.fortuna.ical4j.util.RandomUidGenerator;
+import net.fortuna.ical4j.validate.ValidationException;
```

Comment:
Imports in Maven auslagern

Reply:
Sindse ja. Importieren musste natürlich trotzdem noch, sonst kennt Java die Namespaces nicht. ;-)